src/ui/components/DashKit/plugins/RendererWrapper/RendererWrapper.scss

```diff
@@ -1,7 +1,53 @@
 .dashkit-plugin-container {
+    &__anchor {
+        opacity: 0;
+        right: 100%;
+        padding-right: 4px;
+        height: 100%;
+        float: left;
+
+        &_visible,
+        &:hover,
+        &:focus-visible {
+            opacity: 1;
+        }
+
+        &_absolute {
+            position: absolute;
+        }
+
+        &_size_l {
+            padding-top: 5px;
+            font-size: 24px;
+            line-height: 28px;
+        }
+
+        &_size_m {
+            padding-top: 7px;
+            font-size: 20px;
+            line-height: 24px;
+        }
+
+        &_size_s {
+            padding-top: 7px;
+            font-size: var(--g-text-body-3-font-size);
+            line-height: 24px;
+        }
+
+        &_size_xs {
+            padding-top: 9px;
+            font-size: 15px;
+            line-height: 20px;
+        }
+    }
+
     &__wrapper {
         height: 100%;
 
+        &:hover .dashkit-plugin-container__anchor {
```

Comment:
Class doesn't work because anchor is placed outside the `.dashkit-plugin-container__wrapper`
I suggest putting the anchor classes in a separate file